pictopy/Lib/site-packages/pip/_internal/commands/search.py

```
@@ -0,0 +1,174 @@
+    """
+    The list from pypi is really a list of versions. We want a list of
+    packages with the list of versions stored inline. This converts the
+    list from pypi into one we can use.
+    """
+    packages: Dict[str, "TransformedHit"] = OrderedDict()
+    for hit in hits:
+        name = hit["name"]
+        summary = hit["summary"]
+        version = hit["version"]
+
+        if name not in packages.keys():
```

Ruff (0.11.9)

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)